app/views/events/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= search_form_for @q do |f| %>
+<div class="card">
+  <div class="card-header">
+    <h1>Events</h1>
+  </div>
+  <div class="card-body">
+    <div class="row">
+      <div class="col">
+        <%= f.input :name_cont, label: 'Name' %>
+      </div>
+      <div class="col">
+        <%= f.input :start_time, label: 'Start Time' %>
+      </div>
+      <div class="col">
+        <%= f.input :end_time, label: 'End Time' %>
+      </div>
```

Comment:
These are a) text fields which would be wildly inconvenient and b) don't have ransack predicates so they won't do anything anyway?